example-complete/public/script.js

```diff
@@ -48,7 +48,7 @@ function dec(model) {
 // Update both the counter and the model
 function update(newModel, element) {
   model = newModel;
-  element.innerText = model;
+  element.textContent = model;
```

Comment:
It's referenced in #9 
textContent seems to be more supported in any case, so even without the JSDOM problem we should probably have been using it from the start.